gapic/src/main/com/google/gapid/MainWindow.java

```diff
@@ -91,6 +92,7 @@
   protected final Map<MainTab.Type, Action> viewTabs = Maps.newHashMap();
   protected final Set<MainTab.Type> hiddenTabs = Sets.newHashSet();
   protected Action editCopy;
+  private StatusBar statusBar;
```

Comment:
JFace already has a status bar widget - simply add "addStatusLine()" in the MainWindow constructor.

However, I've not added the status line yet, because if it's empty most of the time, which it looks like it would be in this change, it looks strange.